buildpack.md

```diff
@@ -87,6 +94,11 @@ Buildpack API versions:
  - MUST be in form `<major>.<minor>` or `<major>`, where `<major>` is equivalent to `<major>.0`
  - When `<major>` is greater than `0` increments to `<minor>` SHALL exclusively indicate additive changes
 
+## Types of Buildpacks
```

Comment:
If we add this section I think we need to include `metabuildpack` or `order buildpack` or whatever we call them (`composite buildpack`?).

Reply:
Agree. Can we do that in a different PR?

Reply:
Yes, but probably not a different release (it may be confusing otherwise). I can create an issue.